### DISCUSSION TITLE
What are the reasons and what can I do about it

### DISCUSSION THREAD

**Discussion by KmBooy** [Chia-Network/chia-blockchain#6032] [Q&A]
![image](https://user-images.githubusercontent.com/35790216/119588426-57abfa80-be03-11eb-97a0-70ab30da21bd.png)
The harvester debug and i can't win 5 days. What can i do? I need help

version: windows 10
chia version : 1.1.6

**Comment by tiny0little** [comment]
5 sec in logs is indication that disks are too slow, or maybe you plotting and farming on the same slow disk

**Reply by KmBooy** [reply]
This machine is only farming ，no plotting。why the disks are too slow。How much disk speed is required for farming

**Reply by room101-dev** [reply]
> This machine is only farming ，no plotting。why the disks are too slow。How much disk speed is required for farming

**greater than 5 second delay to 'check' a plot, and your out of the game.**

**### So is your A:/CHIA14 on a network? U can't do this, it must be local**

U can't use network drives, most of us are using SSD drives for the OS these days,

I find that with CHIA, that they just keep increasing the bar, to knock people out of the game, you either step up to the plate, by putting all your 'stuff' on SATA, or you will see this

Also your plots must be attached to the system running the farm, if there on network, then you will hit this limit, 

EVen hpool now has this problem, this error message, so everybody must run all the clients on SSD, and the HDD's must be attached, make sure your not using USB, which is very slow, that's attached, but bad

SATA protocol is old, most use high-speed SAS, that's where all this is going

Long ago CHIA-dev said anybody could plot & farm, on an unused computer, now we're telling you that u must have state of the art enterprise hw, or bugger off.

The farming is cpu-disk intensive just like plotting, I just KILLED a disk yesterday running the 'chia plots check' which is part of the farmer to 'test' the quality of  plots, 

Everything about chia is designed to kill drives, the bar will be continued to be rised, 1 second is coming and your out.

They also keep fine tuning the algo to 'grind' the drives, which causes this delay, It could be selective, like HPOOL users, only time will tell how evil this chia devil really is

**Use old HW, like a $2k USD enterprise 5TB HDD with 1GB DDR5 cache, that kind of cheap around the house drives for your plots.**

**Reply by SackofFluor** [reply]
So far i won 2 block on old HDDs in consumer NAS over network....

**Reply by pluto2021** [reply]
When did that happen? and would be interested to know how many plots you had at the time. Thanks.

**Reply by SackofFluor** [reply]
My first one I won around the 15th of May ( around 600 plots) the 2nd won I got 22nd of May with around 750-800 plots.

2nd one seems to be very lucky.
But I am right now starting to think that most of the net space is ineffective because albight the gui shows "farming" there are errors in the log which prohibit farming.

**Comment by depado** [comment]
This can happen if your drives spins down and needs to spin up again when accessed. It usually takes a few seconds to "wake up" a disk and spin it up again. You should check if there isn't a software associated with your drive that can prevent spin-down or "turn off"/"sleep". 
First of all this will keep your disk healthier (constantly spinning vs spin-down spin-up periodically) and will probably resolve your issue.
